tests/system/api/test_artifacts.py

```diff
@@ -58,3 +61,73 @@ def test_fail_overflowing_artifact(self):
         assert (
             "Failed committing changes to DB" in run["status"]["error"]
         ), "run should fail with a reason"
+
+    def test_model_artifact_tags(self):
+        project_name = "my-test-project"
+        model_name = "test_model"
+        tag_1 = "tag_1"
+        tag_2 = "tag_2"
+        models_path = f"v3io:///projects/{project_name}/artifacts/{model_name}/"
+
+        def _validate_artifact_tags(expected_tags):
```

Comment:
move outside, this test is should be more readable, if you have huge help functions, let them out